lib/debug/session.rb

```diff
@@ -1159,10 +1159,15 @@ def cancel_auto_continue
     end
 
     def show_help arg
+      require_relative "color"
```

Comment:
This is not necessary as the files required by `session.rb` like `tracer` would also load it.

Reply:
require_relative deleted.

Thank you.